src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -8,17 +8,21 @@ import TopicMemberIcon from '../../../assets/images/topic-member-picture.png';
 import {fonts, normalize, normalizeFontSize} from '../../../utils/fonts';
 import {convertString} from '../../../utils/string/StringUtils';
 import {colors} from '../../../utils/colors';
+import {Shimmer} from '../../../components/Shimmer/Shimmer';
 
 const TopicDomainHeader = (props) => {
   const {domain, handleOnMemberPress, hideSeeMember, isFollow, memberCount} = props;
+  const isUserFollow = props?.initialData?.isFollowing ? props.initialData.isFollowing : isFollow;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isUserFollow` variable can be simplified by using logical OR (`||`) instead of the ternary operator. This will make the code more readable.
```diff
- const isUserFollow = props?.initialData?.isFollowing ? props.initialData.isFollowing : isFollow;
+ const isUserFollow = props?.initialData?.isFollowing || isFollow;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->